### DISCUSSION TITLE
Version 3.3 Released!

### DISCUSSION THREAD

**Discussion by gwaldron** [pelicanmapping/osgearth#2053] [Announcements]
This is an incremental bugfix and performance release.
[Release notes are here](http://docs.osgearth.org/en/latest/releasenotes.html).
Enjoy!

**Comment by nebsar** [comment]
Can we still build it with GL2 support?

**Reply by gwaldron** [reply]
Yes, as long as you include GL3 support as well -- osgEarth requires GL 3.3+

**Comment by mmuetzel** [comment]
It looks like the content of the subrepos in `src/third_party` are missing in the tarball. The folders are there. But they are empty.
They were still included in the tarball for 3.2.

**Reply by remoe** [reply]
Here are the docs: https://github.com/gwaldron/osgearth/blob/master/docs/source/build.md

**Reply by mmuetzel** [reply]
So, is building from a release tarball no longer supported? In that case, why are there release tarballs?

**Comment by BAZI-Ilyess** [comment]
Hi, help plz, I can't built osgearth, can some one sent it ti me plz, thank you a lot

**Reply by gwaldron** [reply]
Try installing via vcpkg -- http://docs.osgearth.org/en/latest/install.html
